spec/system/visit_index_page_spec.rb

```diff
@@ -13,4 +14,13 @@
     visit '/'
     expect(page).to have_content 'Manage certificates'
   end
+
+  it 'shows whether certificate is primary or secondary when there are two signing certificates' do
```

Comment:
sorry to be picky, but can we have a test that proves that the newly added cert really becomes a primary... it should be easily done by checking the URLs of the buttons and comparing the IDs